yaksh/models.py

```diff
@@ -2062,6 +2063,39 @@ def get_question_answers(self):
                 }]
         return q_a
 
+    def get_diff_of_consecutive_submissions_for_all_code_question(self):
+        """
+            Return diff of consecutive submissions for all questions
+        """
+        q_a = {} # key : question, value : all submitted answers for this question
```

Comment:
Please write a better, more meaningful variable name.

Reply:
q_a is a dictionary which key is question and value is all submissions

Reply:
q_a is still very short and unclear. Comments help to a certain extent, but then when you use the variable in another file, contributors will still not be able to make out what that variable is for, please use a more clear naming convention.

Discuss or ask if you are unable to decide on a name.

Reply:
done